README.md

```diff
@@ -2398,8 +2405,139 @@ hazelcastClient.getMap('brothersMap').then(function (mp) {
 });
 ```
 
-## 7.9. Monitoring and Logging
+## 7.8. Performance
+
+### 7.8.2. Near Cache
+
+Map entries in Hazelcast are partitioned across the cluster members. Hazelcast clients do not have local data at all. Suppose you read the key `k` a number of times from a Hazelcast client or `k` is owned by another member in your cluster. Then each `map.get(k)` will be a remote operation, which creates a lot of network trips. If you have a data structure that is mostly read, then you should consider creating a local Near Cache, so that reads are sped up and less network traffic is created.
+
+These benefits do not come for free, please consider the following trade-offs:
+
+- If invalidation is enabled and entries are updated frequently, then invalidations will be costly.
+
+- Near Cache breaks the strong consistency guarantees; you might be reading stale data.
+
+Near Cache is highly recommended for data structures that are mostly read.
+
+#### 7.8.2.1. Configuring Near Cache
+
+The following shows the configuration for the Hazelcast Near Cache.
+
+**Declarative:**
+
+```
+{
+    "nearCaches": [
+        {
+            "name": "myDataStructure",
+            "invalidateOnChange": (false|true),
+            "timeToLiveSeconds": (0..Number.MAX_SAFE_INTEGER),
+            "maxIdleSeconds": (0..Number.MAX_SAFE_INTEGER),
+            "inMemoryFormat": "(object|binary|native)",
+            "evictionPolicy": "lru|lfu|random|none",
+            "evictionMaxSize": (0..Number.MAX_SAFE_INTEGER),
+            "evictionSamplingCount": (0..Number.MAX_SAFE_INTEGER),
+            "evictionSamplingPoolSize": (0..Number.MAX_SAFE_INTEGER),
+        }
+    ]
+}
+```
+
+**Programmatic:**
+
+
+Following are the descriptions of all configuration elements:
+
+- `inMemoryFormat`: Specifies in which format data will be stored in your Near Cache. Note that a map’s in-memory format can be different from that of its Near Cache. Available values are as follows:
+  - `BINARY`: Data will be stored in serialized binary format (default value).
+  - `OBJECT`: Data will be stored in deserialized form.
+
+- `invalidateOnChange`: Specifies whether the cached entries are evicted when the entries are updated or removed. Its default value is true.
 
+- `timeToLiveSeconds`: Maximum number of seconds for each entry to stay in the Near Cache. Entries that are older than this period are automatically evicted from the Near Cache. Regardless of the eviction policy used, `timeToLiveSeconds` still applies. Any integer between 0 and `Number.MAX_SAFE_INTEGER`. 0 means infinite. Its default value is 0.
+
+- `maxIdleSeconds`: Maximum number of seconds each entry can stay in the Near Cache as untouched (not read). Entries that are not read more than this period are removed from the Near Cache. Any integer between 0 and `Number.MAX_SAFE_INTEGER`. 0 means `Number.MAX_SAFE_INTEGER`. Its default value is 0.
```

Comment:
`0` means infinite actually(like in timeToLiveSeconds). We represent infinity as MAX_SAFE_INTEGER in implementation but that is an implementation detail. I think users should know this as infinite